src/pages/Quiz.jsx

```diff
@@ -37,6 +38,16 @@ const Quiz = () => {
     }
   };
 
+  const getResponseDone = (clickedCountry) => {
```

Comment:
On fait déjà une requete API où tu peux recup le nom du pays dans la fonction getAnswer.
Tu peux aussi passer par notre fichier JSON "countriesData.json" en faisant:
const clickedCountryData = countryData.find((data) => {
      return data.id === clickedCountry;
    });
clickedCountryData ? setResponseDone(clickedCountryData.name) : setResponseDone("not found");

Ca nous évite de faire des requetes suplementaires